v0/template.json

```diff
@@ -25648,161 +25630,6 @@
               }
             } ]
           },
-          "AzureFunctionDeployStepInfo" : {
```

Comment:
is this getting dropped?

Reply:
This was added twice in the File. So duplicate is being removed.